[CI] DownsampleClusterDisruptionIT testDownsampleIndexWithRollingRestart failing

**Build scan:**
https://gradle-enterprise.elastic.co/s/fhhlgbhrpcmay/tests/:x-pack:plugin:downsample:internalClusterTest/org.elasticsearch.xpack.downsample.DownsampleClusterDisruptionIT/testDownsampleIndexWithRollingRestart
**Reproduction line:**
```
./gradlew ':x-pack:plugin:downsample:internalClusterTest' --tests "org.elasticsearch.xpack.downsample.DownsampleClusterDisruptionIT.testDownsampleIndexWithRollingRestart" -Dtests.seed=975F427D3DCB26B8 -Dtests.locale=el-GR -Dtests.timezone=US/Central -Druntime.java=21
```

**Applicable branches:**
main

**Reproduces locally?:**
Didn't try

**Failure history:**
https://gradle-enterprise.elastic.co/scans/tests?tests.container=org.elasticsearch.xpack.downsample.DownsampleClusterDisruptionIT&tests.test=testDownsampleIndexWithRollingRestart
**Failure excerpt:**
```
com.carrotsearch.randomizedtesting.UncaughtExceptionError: Captured an uncaught exception in thread: Thread[id=515, name=Thread-8, state=RUNNABLE, group=TGRP-DownsampleClusterDisruptionIT]

  at __randomizedtesting.SeedInfo.seed([975F427D3DCB26B8:56BB4E492B562868]:0)

  Caused by: java.lang.AssertionError: timed out waiting for green state

    at __randomizedtesting.SeedInfo.seed([975F427D3DCB26B8]:0)
    at org.junit.Assert.fail(Assert.java:88)
    at org.elasticsearch.test.ESIntegTestCase.ensureColor(ESIntegTestCase.java:1003)
    at org.elasticsearch.test.ESIntegTestCase.ensureGreen(ESIntegTestCase.java:927)
    at org.elasticsearch.test.ESIntegTestCase.ensureGreen(ESIntegTestCase.java:916)
    at org.elasticsearch.xpack.downsample.DownsampleClusterDisruptionIT$Disruptor.run(DownsampleClusterDisruptionIT.java:112)
    at java.lang.Thread.run(Thread.java:1583)

```

Another failure although the stack trace and exception are different to the one above

```
java.lang.AssertionError: (No message provided)	
at __randomizedtesting.SeedInfo.seed([91A71F0132BAB65C:504313352427B88C]:0)	
•••
at org.elasticsearch.xpack.downsample.DownsampleClusterDisruptionIT.assertTargetIndex(DownsampleClusterDisruptionIT.java:384)	
at org.elasticsearch.xpack.downsample.DownsampleClusterDisruptionIT.testDownsampleIndexWithRollingRestart(DownsampleClusterDisruptionIT.java:270)	

```

https://gradle-enterprise.elastic.co/s/onjyc4bx35426/tests/task/:x-pack:plugin:downsample:internalClusterTest/details/org.elasticsearch.xpack.downsample.DownsampleClusterDisruptionIT/testDownsampleIndexWithRollingRestart?top-execution=1

--- a/x-pack/plugin/downsample/src/internalClusterTest/java/org/elasticsearch/xpack/downsample/DownsampleClusterDisruptionIT.java
+++ b/x-pack/plugin/downsample/src/internalClusterTest/java/org/elasticsearch/xpack/downsample/DownsampleClusterDisruptionIT.java
@@ -109,7 +109,7 @@ public class DownsampleClusterDisruptionIT extends ESIntegTestCase {
                     .getNodes()[0].getName();
                 logger.info("Candidate node [" + candidateNode + "]");
                 disruption.accept(candidateNode);
-                ensureGreen(sourceIndex);
+                ensureGreen(TimeValue.timeValueSeconds(60), sourceIndex);
                 ensureStableCluster(cluster.numDataAndMasterNodes(), clientNode);
 
             } catch (Exception e) {
@@ -208,7 +208,6 @@ public class DownsampleClusterDisruptionIT extends ESIntegTestCase {
         assertTargetIndex(cluster, sourceIndex, targetIndex, indexedDocs);
     }
 
-    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/100653")
     public void testDownsampleIndexWithRollingRestart() throws Exception {
         final InternalTestCluster cluster = internalCluster();
         final List<String> masterNodes = cluster.startMasterOnlyNodes(1);

--- a/x-pack/plugin/downsample/src/main/java/org/elasticsearch/xpack/downsample/TransportDownsampleAction.java
+++ b/x-pack/plugin/downsample/src/main/java/org/elasticsearch/xpack/downsample/TransportDownsampleAction.java
@@ -960,11 +960,11 @@ public class TransportDownsampleAction extends AcknowledgedTransportMasterNodeAc
                 .indices()
                 .forceMerge(request, ActionListener.wrap(mergeIndexResp -> actionListener.onResponse(AcknowledgedResponse.TRUE), t -> {
                     /*
-                     * At this point downsampel index has been created
-                     * successfully even force merge fails.
-                     * So, we should not fail the downsample operation
+                     * At this point downsample index has been created
+                     * successfully even if force merge failed.
+                     * So, we should not fail the downsample operation.
                      */
-                    logger.error("Failed to force-merge " + "downsample index [" + downsampleIndexName + "]", t);
+                    logger.error("Failed to force-merge downsample index [" + downsampleIndexName + "]", t);
                     actionListener.onResponse(AcknowledgedResponse.TRUE);
                 }));
         }

